### ISSUE TITLE
F12 (find keyword definition) doesn't work if the keyword is preceded by Run Keyword ...

### ISSUE THREAD

**Issue by bergstenarn** [tomi/vscode-rf-language-server#35]
Imagine the following example where you would like to find the definition of keyword '**Cleanup Test Data**'.

**Run Keyword and Ignore Error`    `Cleanup Test Data**

When placing the cursor within that keyword and pressing F12, VScode displays the following error message:

**No definition found for 'Cleanup'**

However if removing '**Run Keyword ...**' to make the above statement look as follows:

**Cleanup Test Data**

... then VScode successfully finds the definition of '**Cleanup Test Data**'.

**Comment by chenry35** [comment]
Same problem here.

**Comment by loki1600** [comment]
Hi,
I also have this annoying problem :-( 
Any hope for a correction ? or at least a tip to modify ourselves the extension ?
Regards

**Comment by tomi** [comment]
This should be now available in the version 2.3.0. Note that for now only Goto definition is supported. Code completion is not smart enough yet to suggest keywords in these locations.

**Comment by bergstenarn** [comment]
Hi @tomi . Thanks for your effort.
I've now updated this extension to 2.3.0 in VSCode 1.30.2. I'm sorry to say F12 (Go to definition) doesn't work at all anymore, neither with builtin keywords nor with user defined ones ... regardless whether preceded by 'Run Keyword' or not.
I'm convinced you have verified that the 2.3.0 extension update works for you so I'm not sure why it doesn't work for me. Could it be that I'm using .txt extension of my test files? However I have added the following VSCode configuration as explained in the extension documentation so I guess F12 should work also for .txt files:
```
"files.associations": {
  "*.txt": "robot"
}
```
By the way, is it possible to re-open this issue or should I raise a new?
